docs/Measuring_Resonances.md

```diff
@@ -60,6 +58,32 @@ Fritzing wiring diagrams for some of the ADXL345 boards:
 
 ![ADXL345-Rpi](img/adxl345-fritzing.png)
 
+###### Via Raspberry Pi Pico
+
+You may connect the ADXL345 to your Raspberry Pi Pico and then connect the
+Pico to your Raspberry Pi via USB. This makes it easy to reuse the
+accelerometer on other Klipper devices, as you can connect via USB instead
+of GPIO.
+
+Please be particularly careful with your ADXL345. You want one without a
+level shifter and without being forced into I2C mode. For more details,
+see [this discourse post](https://klipper.discourse.group/t/raspberry-pi-pico-adxl345-portable-resonance-measurement/1757).
+
+Note that we do not hook up the INT1 or INT2 pins on the ADXL345.
```

Comment:
Needed?

Reply:
it doesn't hurt or confuse me as a user to have it in there.